cypress/e2e/4-workflow-actions.cy.ts

```diff
@@ -0,0 +1,72 @@
+import { randFirstName, randLastName } from "@ngneat/falso";
+import { DEFAULT_USER_EMAIL, DEFAULT_USER_PASSWORD } from "../constants";
+import { WorkflowPage as WorkflowPageClass } from '../pages/workflow';
+
+const username = DEFAULT_USER_EMAIL;
+const password = DEFAULT_USER_PASSWORD;
+const firstName = randFirstName();
+const lastName = randLastName();
+const newWorkflowName = 'Something else';
+const WorkflowPage = new WorkflowPageClass();
+
+describe('Workflow Actions', () => {
+	beforeEach(() => {
+		cy.signup(username, firstName, lastName, password);
+		cy.on('uncaught:exception', (err, runnable) => {
+			expect(err.message).to.include('Not logged in');
+
+			return false;
+		})
+
+		cy.signin(username, password);
+	});
+
+	it('should be able to save on button slick', () => {
+		WorkflowPage.actions.visit();
+		WorkflowPage.actions.saveWorkflowOnButtonClick();
+		// In Element UI, disabled button turn into spans 🤷‍♂️
+		WorkflowPage.getters.saveButton().should('match', 'span');
+	});
+
+	it('should save workflow on keyboard shortcut', () => {
+		WorkflowPage.actions.visit();
+		WorkflowPage.actions.saveWorkflowUsingKeyboardShortcut();
+		WorkflowPage.getters.saveButton().should('match', 'span');
+	});
+
+	it('should not be able to activate unsaved workflow', () => {
+		WorkflowPage.actions.visit();
+		WorkflowPage.getters.activatorSwitch().find('input').first().should('be.disabled');
+	});
+
+	it('should not be able to activate workflow without trigger node', () => {
+		WorkflowPage.actions.visit();
+		// Manual trigger is not enough to activate the workflow
+		WorkflowPage.actions.addTriggerNode('n8n-nodes-base.manualTrigger');
```

Comment:
I'm not sure if using nodeType is ideal here. Because the target of the e2e test should be to test the interface, and the user would usually type "Manual trigger" rather than look for node items by their ID. I've prepared a feature object for the NodeCreator that does this, it just needs to be reviewed (Ivan's on it) perhaps you could then merge it into your branch as there will be some conflicts since you implement some of the node creator actions here.

Reply:
Ok, used that. My thinking here was to avoid using labels/display names as much as possible since they tend to change but makes sense in context of e2e testing